src/components/EmailNotifications/EmailNotificationsForm.js

```diff
@@ -279,7 +272,7 @@ function LegalTermsAndPolicy({ termsAccepted, setTermsAccepted }) {
             text-align: left;
           `}
         >
-          {text}
+          By continuing with your email, you agree to Aragon Court{' '}
```

Comment:
```suggestion
          By continuing with your email, you agree to Aragon Court's{' '}
```

Having to duplicate this text (and make sure it's the same across all the usages) might be a good indicator to make a shared component out of this. Even better if it's simple!